cli/determined_cli/checkpoint.py

```diff
@@ -72,11 +72,19 @@ def list(args: Namespace) -> None:
     ).json()
     searcher_metric = r["metric_name"]
 
-    headers = ["Trial ID", "Step ID", "State", "Validation Metric", "UUID", "Resources", "Size"]
+    headers = [
+        "Trial ID",
+        "# of Batches",
+        "State",
+        "Validation Metric",
+        "UUID",
+        "Resources",
+        "Size",
+    ]
     values = [
         [
             c["trial_id"],
-            c["step_id"],
+            c["step"]["total_batches_processed"] + c["step"]["num_batches"],
```

Comment:
For steps, total_batches_processed are the batches processed before that step, and num_batches are the batches in that steps, so to know how many were done at that checkpoint we have to add them. In hindsight, this variable should've been prior_batches_processed, but it's all over the harness and everywhere else now and would be way too much effort to change.